include/oboe/AudioStream.h

```diff
@@ -158,9 +158,30 @@ class AudioStream : public AudioStreamBase {
 
     virtual int64_t getFramesRead() const { return mFramesRead; }
 
+    /**
+     * Calculate the latency of a stream based on getTimestamp().
+     *
+     * Latency is the time it takes for a given frame to travel from the
+     * app to the edge of the device or vice versa.
+     *
+     * Note that the latency of an OUTPUT stream will increase when you write data to it
```

Comment:
Do you mean the results provided by this function, or the actual latency (e.g. measured using a dedicated rig)?

Reply:
Both. The actual output latency is a sawtooth function that jumps up when you write data then slowly falls as the data is consumed.